source/features/rgh-linkify-features.tsx

```diff
@@ -22,7 +30,7 @@ function initTitle(): void {
 }
 
 function init(): void {
-	for (const possibleMention of select.all('.js-comment-body code, .markdown-body li code')) {
+	for (const possibleMention of select.all(':is(.js-comment-body, .markdown-body li, .markdown-title) code, code .markdown-title')) {
```

Comment:
About the new convoluted selectors: https://github.com/refined-github/refined-github/issues/4135#issuecomment-830171333

Reply:
Also one selector per line would be good